tests/test_formula_parser.py

```diff
@@ -339,6 +339,14 @@ def test_decimal(self):
             answer=[1 + 0.009 / 0.099 * 0.009 - 1],
         )
 
+    def test_order_of_operations(self):
+        _test_equation(equation="(2^(-1))", variables={"a1": [1]}, answer=[0.5])
+        _test_equation(equation="((2^(-1))-1)", variables={"a1": [1]}, answer=[-0.5])
+        _test_equation(equation="(2^((-1)-1))", variables={"a1": [1]}, answer=[0.25])
+        _test_equation(equation="(2^1-1)", variables={"a1": [1]}, answer=[1])
+        _test_equation(equation="(2^(-1)-1)", variables={"a1": [1]}, answer=[-0.5])
```

Comment:
This is the test case that is failing. It is doing `2 ^((-1) - 1)` instead of  `2^-2`